openshift_scalability/nodeVertical.sh

```diff
@@ -19,8 +41,15 @@ clean() { echo "Cleaning environment"; oc delete project clusterproject0; }
 # sleeping to gather some steady-state metrics, pre-test
 long_sleep
 
-# loading cluster based on yaml config file
-./cluster-loader.py --file=$MY_CONFIG
+# Run the test
+if [ "$TYPE" == "golang" ]; then
+  golang_clusterloader
+elif [ "$TYPE" == "python" ]; then
+  python_clusterloader
+else
+  echo "$TYPE is not a valid option, available options: golang, python"
+  exit 1
+fi
 
 # sleeping again to gather steady-state metrics after environment is loaded
 long_sleep
```

Comment:
I believe there is no need to call clean for golang - the e2e framework should clean up the project.